assets/src/scss/5-pages/_guide.scss

```diff
@@ -1,5 +1,6 @@
 .guide-content {
-  @include rem(margin-top, 50px);
+  margin-top: 0;
+  //@include rem(margin-top, 50px);
```

Comment:
will clean up comments when done. Commented because not sure if I need to revert it.